pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -40,6 +40,10 @@ const (
 	IPSecDirOut     IPSecDir = "IPSEC_OUT"
 	IPSecDirBoth    IPSecDir = "IPSEC_BOTH"
 	IPSecDirOutNode IPSecDir = "IPSEC_OUT_NODE"
+
+	// ipSecXfrmMarkSPIShift defines how many bits the SPI is shifted when
+	// encoded in a XfrmMark
+	ipSecXfrmMarkSPIShift = 12
```

Comment:
Commit `ipsec: add ipSecXfrmMark{Set,Get}SPI helpers` is missing a reference to the upstream commit.